optimization/generate_profile.py

```diff
@@ -51,14 +51,11 @@ def calculate_fit(v_profile, e_profile, max_time=450, dist_step=30):
     energy = car.energy_used(v_profile, e_profile, distance=dist_step)
     # We ignore the first point since v is 0 at the start
     time = [dist_step / v for v in v_profile[1:]]
-    if sum(time) > max_time:
-        fit = -1 # TODO: allow smart usage of too slow runs
-    else: 
-        fit = energy
+    fit = energy
     return [fit, sum(time)]
 
 
-def generate_new_profile(v_profile, e_profile, dist_step=30, min_v = 5):
+def generate_new_profile(v_profile, e_profile, dist_step=30, min_v=4):
```

Comment:
need to change back min_v

Reply:
Min_V should also be a parameter passed by the user